tests/integration/scenario/entities.py

```diff
@@ -6,14 +6,18 @@
 
 
 class PetType(Enum):
-    """ Enum that describes the type of pet """
+    """Enum that describes the type of pet"""
+
     DOG = "dog"
     CAT = "cat"
     FISH = "fish"
 
 
 class Pet(SQLModelEntity, table=True):
-    """ Pet model """
+    """Pet model"""
+
+    id: int = Field(index=True, default=None, primary_key=True)
```

Comment:
shouldn't this be already covered by the inheritance of `SQLModelEntity`?

Reply:
Yes it is, just explicitness